src/providers/__tests__/ProfileModalProvider.test.js

```diff
@@ -0,0 +1,54 @@
+import 'react-native';
+import * as React from 'react';
+import { ProfileModalProvider } from '../ProfileModalProvider';
+import ProfileModal from '../../components/shared/ProfileModal';
+
+// Note: test renderer must be required after react-native.
+import renderer from 'react-test-renderer';
+import { shallow, render, mount } from 'enzyme';
+
+describe('[ProfileModalProvider] rendering test', () => {
+  const wrapper = shallow(<ProfileModalProvider/>);
+  it('[ProfileModalProvider] renders as expected', () => {
+    expect(wrapper).toMatchSnapshot();
+  });
+});
+
+describe('[ProfileModalProvider] interaction', () => {
+  let props;
+  let providerWrapper;
+  let modalWrapper;
+  const friend = {
+    uid: '2',
+    displayName: 'geoseong',
+    photoURL: null,
+    statusMsg: 'healthy',
+    isOnline: '',
+    friends: '',
+    Chatrooms: '',
+    created: '',
+    updated: '',
+  };
+
+  beforeEach(() => {
+    props = {
+      navigation: {
+        navigate: jest.fn(),
+      },
+      state: {
```

Comment:
Isn't `state` and `props` different?

Reply:
oh, this code is not necessary. I will remove it on next commit.